resources/digital_systems.md

```diff
@@ -0,0 +1,145 @@
+
+- Knowledge on how to read Venn Diagrams
+
+- Knowledge of Number Systems and Conversion between them
+
+---
+
+## Classification of Codes
+
+**Code:** When numbers or letters are represented by a specific group of symbols, we can see that the numbers or letters are **encoded**. The group of symbols are called "Code". 
+
+```
+Codes = Group of Symbols
+```
+
+Binary Codes can described as being part of one or more of the following families:
+
+| Weighted Codes                                                                                    | Non-Weighted Codes                                                                                   | Reflective Codes                                                                                            | Sequential Codes                                                                                                             | Alphanumeric Codes                                                                                                  | Error Detecting  & Corrector Codes                                                                                         |
+|---------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------|
+| There is a weight associated with each digit, dependant on the number's position in the sequence. | Digit Value is independent of a number's position in the sequence. Gray Code is one example of this. | _k-th_ smallest number is the complement of  the _K-th_ largest number. Look up the 5211 code or 2421 code. | For every  subsequent number, the binary representation only differs by one digit. Excess-3 and 8421 are codes of this type. | The family of binary codes used to represent alphanumeric data. The primordial example would be the ASCII encoding. | The family of binary encoding is meant to detect and or correct failed data transmission.  Parity Checking is one example. |
+
+---
+
+## Multiplexer
+
+A multiplexer is a combinational circuit that selects one of many input lines and directs it to a single output line. The selection of a particular input line is controlled by a set of selection lines. Normally, there are $2^𝑛$ input lines and 𝑛 selection lines whose bit combinations determine which input is selected.
+
+An example of a 4 to 1 multiplexer is provided below.
+
+![4 to 1 Multiplexer.png](/resources/pictures/4%20to%201%20Multiplexer.png)
```

Comment:
I believe the image contains snippets from a paid textbook. Do not use / re-uploaded images which likely are copyrighted. If the image is too good, you can reference the image via a website image link so that there is attribution for now.

Reply:
Same as comment on lines +160 to +163.

Reply:
remade image independently.